packages/esm-service-queues-app/src/patient-queue-header/patient-queue-header.component.tsx

```diff
@@ -48,8 +56,8 @@ const PatientQueueHeader: React.FC<{ title?: string }> = ({ title }) => {
             <label className={styles.view}>{t('view', 'View')}:</label>
             <Dropdown
```

Comment:
If `isLoading` is true, then we will show [DropdownSkeleton](https://react.carbondesignsystem.com/?path=/story/components-dropdown--skeleton).

If there is an error, we can show one option, i.e. All, and show a Notification with the error message.